libcaf_core/caf/serialization.test.cpp

```diff
@@ -729,11 +746,13 @@ OUTLINE("serializing and then deserializing primitive values") {
     | binary_serializer   | u32    | 123456        |
     | binary_serializer   | u64    | 123456789     |
     | binary_serializer   | ld     | 123.5         |
+    | binary_serializer   | f      | 3.14          |
     | binary_serializer   | real   | 12.5          |
     | binary_serializer   | string | Hello, world! |
     | binary_serializer   | u16str | Hello, world! |
     | binary_serializer   | u32str | Hello, world! |
     | binary_serializer   | vector | 1, 42, -31    |
+    | binary_serializer   | vector_bool | true,true,false,false,true,true,true,false |
```

Comment:
That's a bit unwindy, abbreviating it indents nicer and is less verbose:

```suggestion
    | binary_serializer   | vbool | true,true,false,false,true,true,true,false |
```

Is there a particular reason why this has 8 values? Could just have two to avoid this overly long line.

Reply:
The serializer and deserializer use a switch case to serialize/deserialize an array into a single byte. I have added 8 entries to ensure that all conditions are being handled properly. Please let me know if it will be okay to skip it.